### DISCUSSION TITLE
Presaved filters as multi list page.

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#17055] [Feature Requests]
Hi:

I am thinking of a new feature:   save some filters as a list, then the user can choose an item to show the record list.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
